### DISCUSSION TITLE
Banner Component

### DISCUSSION THREAD

**Discussion by tveinfeld** [Vonage/vivid#799] [New Developments]
Let's discuss our Banner component:

- Usage scenarios
- Functionality
- Interface choices

**Comment by tveinfeld** [comment]
Do you think the Banner's primary slot should contain any content, or should we separate text from buttons and other components using specific interface.

**Reply by yinonov** [reply]
I suggest adding a `label` prop as "1st aid" and later on consider slotting if need
